### ISSUE TITLE
Test out designs for other 'layers' that can be mixed and matched with Park Tiles

### ISSUE THREAD

**Issue by makella** [nationalparkservice/park-tiles#149]
@nateirwin and I have discussed for quiet some time the idea of designing separate layers that can be mixed and matched with Park Tiles.  Some examples include landcover, other public lands, landforms, etc. It would be great if we could prototype out one of these to see what the data requirements are and how something like this could be exposed in Builder.

The idea here is to leave Park Tiles as a canvas to overlay information but if a user wishes to be able to add more detail to the map based on a given project.  This is what we are currently doing with Points of Interest from Places.

**Comment by cwlawlis802** [comment]
Is this something we are still looking to pursue @nateirwin?

**Comment by nateirwin** [comment]
Yes, for sure! This is actually part of the exercise we're currently going through related to rethinking how we architect Park Tiles and its derivatives.
